frontend/src/components/browse-sightings-page/MapPage.tsx

```diff
@@ -0,0 +1,43 @@
+import { Sighting } from "../../clients/apiClient";
+// import { SlippyMap, Marker, Label, InfoBox } from "react-slippy-map";
+import GoogleMapReact from "google-map-react";
+import { Marker } from "./marker";
+import React from "react";
+import { map } from "leaflet";
+import "./mapPage.scss";
+
+interface MapPageProps {
+  sightingList: Sighting[];
+}
+
+export const MapPage: React.FC<MapPageProps> = ({ sightingList }) => {
+  const points = sightingList.map((sighting, index) => {
+    return {
+      id: sighting.id,
+      title: sighting.seenBy,
+      lat: sighting.latitude,
+      lng: sighting.longitude,
+    };
+  });
+
+  return (
+    <div style={{ height: "100vh", width: "100vh" }}>
+      ;
+      <GoogleMapReact
+        className="map"
+        bootstrapURLKeys={{
+          // remove the key if you want to fork
+          key: "AIzaSyAiX_Ww0KPPTu7QZ2oODx5Jah9Ky1F7QTU",
+          language: "en",
+          region: "US",
+        }}
+        defaultCenter={{ lat: 51.506, lng: -0.169 }}
+        defaultZoom={15}
+      >
+        {points.map(({ lat, lng, id, title }) => {
+          return <Marker text={title} key={id} lat={lat} lng={lng} />;
+        })}
```

Comment:
I'm really confused by this, since you don't use the `lat` and `lng` fields in your `Marker` component... so how does the map know where to place them? You have tested this with real sightings, right?

Reply:
As discussed on discord, I think we have to leave this in for the magic to work under the hood